evalml/automl/automl_search.py

```diff
@@ -69,6 +56,22 @@
 logger = get_logger(__file__)
 
 
+def get_default_primary_search_objective(problem_type):
+    """Get the default primary search objective for a problem type.
+
+    Arguments:
+        problem_type (str or ProblemType): problem type of interest.
+
+    Returns:
+        ObjectiveBase: primary objective instance for the problem type.
+    """
+    problem_type = handle_problem_types(problem_type)
+    objective_name = {'binary': 'Log Loss Binary',
+                      'multiclass': 'Log Loss Multiclass',
+                      'regression': 'R2'}[problem_type.value]
+    return get_objective(objective_name, return_instance=True)
```

Comment:
Awesome.

I suggest we move this to `evalml/automl/utils.py`. And include it in `evalml/automl/__init__.py`

Reply:
Done!